### DISCUSSION TITLE
Wordle

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#633] [General]
Do you guys know about the most popular game this year?
How much time do you spend playing it?

**Comment by MirkosMangle** [comment]
Not a big fan of word games to be honest.
Prefer some horrors or action games

**Comment by hernessteve** [comment]
Huge fan here! Can spend hours and hours playing Wordle, Scrabble or Words with the friends.
For the last one we actually found a very cool cheating tool [https://word-finder.com/words-with-friends-cheat-tool/](url) which everyone can use 5 times a day. It doesn't make it less fun but probably the opposite.
